variables.tf

```
@@ -101,3 +101,9 @@ variable "extra_accounts" {
   type        = list(string)
   default     = []
 }
+
+variable "ssl_secret_key" {
+  description = "Custom secret key for ca certificate."
```

At least the description could be more explicit:
```suggestion
  description = "Key of the `argocd-tls` secret containing a CA certificate to be accepted by Argo CD."
```

---

I agree with you, I'll update the PR with your proposal :wink: